superset/dashboards/commands/importers/v0.py

```diff
@@ -131,7 +131,7 @@ def alter_positions(
         for value in position_json:
             if (
                 isinstance(value, dict)
-                and value.get("meta")
+                and value.get("type") == "CHART"
```

Comment:
This is needed given how as part of the migration we're temporarily storing the old `chartId` in the metadata block for the new markdown elements (which serve as a replacement for the filter-box charts) as a way to revert.

Granted this is bastardizing the metadata somewhat, but it does seem prudent to also ensure that the type is `CHART` prior to fetching the `chartId`.